contents/scalability/scalable-development.md

```diff
@@ -15,12 +15,33 @@ So what is scalability? Scalability is the capacity to which a system can cope w
 
 Let's say you have a server running on a single host, but due to the growing number of users, it has been slowing down. What are the different ways you can speed it up?
 
-### Get Better Hardware
+### Horizontal vs Vertical Scaling
+
+#### Horizontal Scaling
+
+Buy more machines to host your server. This is called horizontal scaling. This method can help improve scalability by a huge factor, as each additional unit of processing power and RAM costs the same, no matter how many machines you buy. As this is a promising way to increase scalability, it is no surprise why large services such as Google and Facebook use thousands of computer servers just to provide services to all their users.
```

Comment:
> Buy more machines to host your server. This is called horizontal scaling.

Unnecessary verbosity :p. "Horizontal scaling is simply buying more machines to host your server."